src/__tests__/01b-create-users.ts

```diff
@@ -33,6 +37,16 @@ it("add user0/funder/Dealer", async () => {
 
 const username = "user0"
 
+const generateTokenHelper = ({ secret }) => {
+  const generateTokenResult = generateToken(secret)
+  expect(generateTokenResult).toBeTruthy()
+  if (generateTokenResult) {
+    return generateTokenResult.token
+  } else {
+    fail("generateToken returned null")
```

Comment:
`fail` is part of the `jest` toolbox?

Reply:
`fail` is from jasmine which jest uses internally afaik